### DISCUSSION TITLE
Call Zk Alias From Neovim

### DISCUSSION THREAD

**Discussion by shfattig** [zk-org/zk-nvim#178] [Help] [answered]
I have a `zk daily` alias that I often want to pull up while inside my editor already. Is there a mechanism for running these kinds aliases using this plugin? Or is there otherwise a graceful way to open up my daily note inside the current editor process? (I can use ZkNotes but it's less ergonomic)

**Comment by zalegrala** [comment, marked as answer]
I have this in my `keys` under lazy.nvim for a similar purpose.

```lua
      { "<leader>zw", ':lua require("zk.commands").get("ZkNew")({ dir = "journal/weekly" })<CR>' },
```

**Comment by tjex** [comment]
That would be the way 👍
You would then want to declare a directory under the `daily.group` in your conf, so then `ZkNew` would name the file / format it as you like.

Or specify a daily group, which is defined in the `.zk/config.toml`.

```lua
require("zk.commands").get("ZkNew")({ dir = diary_dir, group = "daily"  })
```

I've [extended my command a bit](https://git.sr.ht/~tjex/dotfiles/tree/mac/item/.config/nvim/lua/tjex/plugins/zk.lua#L27) to add a subtitle to daily notes.
